Source/WTF/wtf/PlatformHave.h

```diff
@@ -1265,6 +1265,14 @@
 #define HAVE_APPLE_PUSH_SERVICE_URL_TOKEN_SUPPORT 1
 #endif
 
+#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 130000) \
+    || (PLATFORM(MACCATALYST) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 160000) \
+    || (PLATFORM(IOS) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 160000) \
+    || (PLATFORM(APPLETV) && __TV_OS_VERSION_MAX_ALLOWED >= 160000) \
+    || (PLATFORM(WATCHOS) && __WATCH_OS_VERSION_MIN_REQUIRED >= 90000)
+#define HAVE_AVIF 1
+#endif
+
 #if !PLATFORM(IOS_FAMILY)
 #define HAVE_MEDIA_VOLUME_PER_ELEMENT 1
 #endif
```

Comment:
(not sure where to leave this comment) It's pretty surprising that no sandbox changes are necessary. The commit message doesn't address this. Why do these images work without any sandbox changes?

Reply:
You are right I added the needed code to enable software image decoding for AVIF in `WebProcess::platformInitializeWebProcess()`